split-proposal-pseudocode.md

```diff
@@ -0,0 +1,10 @@
+# Split Proposal
+
+## High level spec
+
+- any Nouner can create a new split proposal, as long as there isn't already an active one.
```

Comment:
from @davidbrai: we don't need to maintain state of an 'active' proposal, just let people escrow